tests/cli/RunCest.php

```diff
@@ -633,4 +635,262 @@ public function runBootstrapInSuiteConfig(\CliGuy $I)
         $I->seeInShellOutput("OK (");
     }
 
+
+    protected function htmlReportRegexCheckProvider()
+    {
+      return [
+        [
```

Comment:
All data items could be named by using string keys in top level array.
`'twoCommentStepsInARow' =>[`

Reply:
No, as aforementioned: IIRC named datasets in dataprovider also does not work (anymore) as documented; as a workaround there is e.g. overridden `__toString()` in https://github.com/Codeception/Codeception/pull/5690/commits/dde81b809d99333504e49fb3c146c5cbb2e915ad

Reply:
Ah, named datasets work properly in Test format only.
One option would be to move your test to a new Test file.
But it would be the best to have separate methods for each case.

Reply:
Why is that better? Then I would need to duplicate the code of `RunCest.runHtmlCheckReport` for each test case, or have another helper method that is called by each of these separate method (for each case)? Why not just leave the parameterization/`@dataProvider` usage as is?

I find the current test execution log output really fine:
```
...
RunCest: Execute tests with html output and check html (CodeceptionIssue4413Cest:twoCommentStepsInARow) (0.14s)
RunCest: Execute tests with html output and check html (CodeceptionIssue4413Cest:twoCommentStepsInARowViaPageObjectActor) (0.13s)
RunCest: Execute tests with html output and check html (CodeceptionIssue4413Cest:twoCommentStepsWithOneSubStepInBetween) ...
```

(Just to avoid a possible misunderstanding: The aforementioned refactoring -- extraction of `Codeception\Test\Descriptor.getTestCaseNameAsString()` -- is not needed for this parameterization in `RunCest`, but for assertion of better test case names of `CodeceptionIssue4413Cest` written into HTML report in this nice style.)